src/operations/filter.js

```diff
@@ -0,0 +1,45 @@
+const stockFilter = (products, includeOutStock) => {
+  const filteredProducts = includeOutStock
+    ? products
+    : products.filter((product) => product.inStock === true);
+  return filteredProducts;
```

Comment:
return directly instead of declaring new variable and returning, unless you have any specific use case.